app/models/proxy.rb

```diff
@@ -64,16 +64,16 @@ class Proxy < ApplicationRecord # rubocop:disable Metrics/ClassLength
   validates :credentials_location, inclusion: { in: %w[headers query authorization], allow_nil: false }
 
   validates :error_status_no_match, :error_status_auth_missing, :error_status_auth_failed, :error_status_limits_exceeded,
-                            numericality: { greater_than_or_equal_to: 200, less_than: 600 }
+            numericality: { greater_than_or_equal_to: 200, less_than: 600 }
 
   validates :auth_app_id, :auth_app_key, :auth_user_key, :secret_token,
-                      format: { with: APP_OR_USER_KEY, allow_nil: false }
+            format: { with: APP_OR_USER_KEY, allow_nil: false }
 
   validates :error_headers_auth_failed,
-                      :error_headers_auth_missing, :error_headers_no_match,
-                      :error_auth_failed, :error_auth_missing, :error_no_match,
-                      :error_headers_limits_exceeded, :error_limits_exceeded,
-                      format: { with: HTTP_HEADER }
```

Comment:
please avoid just formatting changes even if formatting is not ideal to make review easier and git blame easier to follow